javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,27 @@
+//find shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+let shortestWordLength= danishWords[0].length;
+danishWords.forEach(shortestWordinArray)
+function shortestWordinArray(wordsArray){   
+    if(wordsArray.length < shortestWordLength){
+        shortestWordLength=wordsArray.length;
+        console.log('shortest word "'+wordsArray +'" length :'+shortestWordLength);   
+    } 
+}
+//Find and count the Danish letters
+const danishString = "Jeg har en blå bil";
+findDanishAlphabets(danishString); // returns {total: 1, å: 1}
+
+const danishString2 = "Blå grød med røde bær";
+findDanishAlphabets(danishString2); // returns {total: 4, æ: 1, ø: 2, å: 1}
+
+function findDanishAlphabets(Sentance){
```

Comment:
I would call this logDanishLetters.

Its parameter starts with a capital letter, which can be a bit confusing, because that is usually used for types, and this is not a type.